cmd/minikube/cmd/start.go

```diff
@@ -85,6 +85,7 @@ const (
 	vsockPorts            = "hyperkit-vsock-ports"
 	gpu                   = "gpu"
 	embedCerts            = "embed-certs"
+	noVTXCheck            = "virtualbox-no-vtx-check"
```

Comment:
Looks great. Do you mind altering this to a more general flag name, such as --no-vtx-check? 

I ask because a feature such as this will likely be applicable to other virtualization engines in the future, such as kvm2. I'd rather not add a flag per-VM manager if I could avoid it.

Reply:
Sure thing!